vcs-utils/src/main/java/com/arcbees/vcs/bitbucket/model/BitbucketCommitStatus.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2014 ArcBees Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.arcbees.vcs.stash.model;
```

Comment:
package is wrong, it should be `com.arcbees.vcs.bitbucket.model`. This might be causing the duplicate class error